.eslintrc

```diff
@@ -2,7 +2,8 @@
   "plugins": ["prettier"],
   "extends": "eslint:recommended",
   "env": {
-	"node": true
+	"node": true,
+	"es6": true
```

Comment:
Yep! Based on the other new language features you're already using, I don't think this changes the oldest supported version of node.js.